<!DOCTYPE html>
<html>

<head>
    <meta charset="UTF-8">
    <title>1&period; Metabolism pathway</title>
    <style>
    </style>

    <link rel="stylesheet"
        href="https://cdn.jsdelivr.net/gh/Microsoft/vscode/extensions/markdown-language-features/media/markdown.css">
    <link rel="stylesheet"
        href="https://cdn.jsdelivr.net/gh/Microsoft/vscode/extensions/markdown-language-features/media/highlight.css">
    <style>
        body {
            font-family: -apple-system, BlinkMacSystemFont, 'Segoe WPC', 'Segoe UI', system-ui, 'Ubuntu', 'Droid Sans', sans-serif;
            font-size: 14px;
            line-height: 1.6;
        }
    </style>
    <style>
        .task-list-item {
            list-style-type: none;
        }

        .task-list-item-checkbox {
            margin-left: -20px;
            vertical-align: middle;
        }
    </style>



</head>

<body class="vscode-body vscode-light">
    <h1 id="1-metabolism-pathway">1. Metabolism pathway</h1>
    <p><strong>Provided by Gao Senquan(2019013224), Chen Shurui(2019013222), Diao Tongxin(2019013240) and Lu
            Minsi(2019080058) in THU. We've also push it to <code>Github</code> repository
            (Available at <a
                href="https://github.com/maxwell-gao/CH_metabolism_path">https://github.com/maxwell-gao/CH_metabolism_path</a>)</strong>
    </p>
    <h2 id="11-preface">1.1. <strong>Preface</strong></h2>
    <p>The majority of our figures are generated by [<code>Escher</code>] (<a
            href="https://github.com/zakandrewking/escher/">https://github.com/zakandrewking/escher/</a>),
        [<code>COBRApy</code>] (<a href="https://opencobra.github.io/cobrapy/">https://opencobra.github.io/cobrapy/</a>)
        and models in [<code>Jupyter Lab</code>] (<a href="https://jupyter.org/">https://jupyter.org/</a>). The
        <code>BioRender</code> is also used for figure creating.</p>
    <p>We use <code>COBRA</code> as an analyzing tool based on omics data flux to reveal how the metabolism flux
        actually works in the living body.</p>
    <p>This comprehensive homework is composed of three major parts: the source code we use, the figures we draw with
        various tools (we believe that you may find them so different from each other soonðŸ˜‚), and the document you are
        reading now (in the HTML form and the markdown form).</p>
    <p>We aim to provide a creative form for biochemistry reviewing and understanding though we know thatâ€™s a hard work.
    </p>
    <ul>
        <li><a href="#1-metabolism-pathway">1. Metabolism pathway</a>
            <ul>
                <li><a href="#11-preface">1.1. <strong>Preface</strong></a></li>
                <li><a href="#12-metabolism-overview">1.2. <strong>Metabolism Overview</strong></a>
                    <ul>
                        <li><a href="#121-the-core-metabolism">1.2.1. The core metabolism</a></li>
                    </ul>
                </li>
                <li><a href="#13-central-carbon-metabolism">1.3. <strong>Central carbon Metabolism</strong></a>
                    <ul>
                        <li><a href="#131-flux-of-central-carbon">1.3.1. Flux of central carbon</a></li>
                        <li><a href="#132-emp-tca-and-ppp">1.3.2. EMP, TCA and PPP</a></li>
                        <li><a href="#133-glycolysis-and-glycolytic-enzymes">1.3.3. Glycolysis and Glycolytic
                                Enzymes</a></li>
                        <li><a href="#134-glycogen-degradation">1.3.4. Glycogen Degradation</a></li>
                        <li><a href="#135-krebs-cycle">1.3.5. Krebs Cycle</a></li>
                        <li><a href="#136-pentose-phosphate-pathway">1.3.6. Pentose Phosphate Pathway</a></li>
                        <li><a href="#137-oxidative-phosphorylation">1.3.7. Oxidative Phosphorylation</a></li>
                        <li><a href="#138-warburg-effect">1.3.8. Warburg Effect</a></li>
                        <li><a href="#139-the-cori-cyle-an-anaerobic-metabolic-method-for-glucogenesis">1.3.9. The Cori
                                Cyle: An Anaerobic Metabolic Method for Glucogenesis</a></li>
                    </ul>
                </li>
                <li><a href="#14-fatty-acid-metabolism">1.4. <strong>Fatty acid Metabolism</strong></a>
                    <ul>
                        <li><a href="#141-fatty-acid">1.4.1. Fatty Acid</a></li>
                    </ul>
                </li>
                <li><a href="#15-amino-acid-metabolism">1.5. <strong>Amino acid Metabolism</strong></a>
                    <ul>
                        <li><a href="#151-amino-acid-catabolism-and-urea-cycle">1.5.1. Amino Acid Catabolism and Urea
                                Cycle</a></li>
                        <li><a href="#152-metabolism-of-amino-acid-carbon-skeletons">1.5.2. Metabolism of Amino Acid
                                Carbon Skeletons</a></li>
                    </ul>
                </li>
                <li><a href="#16-nucleotide-metabolism">1.6. <strong>Nucleotide Metabolism</strong></a>
                    <ul>
                        <li><a href="#161-nucleotide-metabolism">1.6.1. Nucleotide Metabolism</a></li>
                    </ul>
                </li>
                <li><a href="#17-reference">1.7. <strong>Reference</strong></a></li>
            </ul>
        </li>
    </ul>
    <h2 id="12-metabolism-overview">1.2. <strong>Metabolism Overview</strong></h2>
    <h3 id="121-the-core-metabolism">1.2.1. The core metabolism</h3>
    <p><img src="GIF/core.gif" alt=""></p>
    <p><object width="75%" height="800px" data="Core.html"></object></p>
    <h2 id="13-central-carbon-metabolism">1.3. <strong>Central carbon Metabolism</strong></h2>
    <h3 id="131-flux-of-central-carbon">1.3.1. Flux of central carbon</h3>
    <p><object width="75%" height="1000px" data="CarbonMeta.html"></object></p>
    <h3 id="132-emp-tca-and-ppp">1.3.2. EMP, TCA and PPP</h3>
    <p><object width="75%" height="500px" data="EMP_TCA_PPP.html"></object></p>
    <h3 id="133-glycolysis-and-glycolytic-enzymes">1.3.3. Glycolysis and Glycolytic Enzymes</h3>
    <p><img src="Figures/Glycolysis%20and%20Glycolytic%20Enzymes.png" alt="Glycolysis and Glycolytic Enzymes"></p>
    <h3 id="134-glycogen-degradation">1.3.4. Glycogen Degradation</h3>
    <p><img src="Figures/Glycogen%20degradation.png" alt="Glycogen Degradation"></p>
    <h3 id="135-krebs-cycle">1.3.5. Krebs Cycle</h3>
    <p><img src="Figures/Krebs%20Cycle.png" alt="Krebs Cycle"></p>
    <h3 id="136-pentose-phosphate-pathway">1.3.6. Pentose Phosphate Pathway</h3>
    <p><img src="Figures/Pentose%20Phosphate%20Pathway.png" alt="Pentose Phosphate Pathway"></p>
    <h3 id="137-oxidative-phosphorylation">1.3.7. Oxidative Phosphorylation</h3>
    <p><img src="Figures/Oxidative%20Phosphorylation.png" alt="Oxidative Phosphorylation"></p>
    <h3 id="138-warburg-effect">1.3.8. Warburg Effect</h3>
    <p><img src="Figures/Warburg%20Effect.png" alt="Warburg Effect"></p>
    <h3 id="139-the-cori-cyle-an-anaerobic-metabolic-method-for-glucogenesis">1.3.9. The Cori Cyle: An Anaerobic
        Metabolic Method for Glucogenesis</h3>
    <p><img src="Figures/The%20Cori%20Cycle_%20An%20Anaerobic%20Metabolic%20Method%20for%20Gluconeogenesis.png"
            alt="The Cori Cycle: An Anaerobic Metabolic Method for Gluconeogenesis"></p>
    <h2 id="14-fatty-acid-metabolism">1.4. <strong>Fatty acid Metabolism</strong></h2>
    <h3 id="141-fatty-acid">1.4.1. Fatty Acid</h3>
    <p><img src="GIF/FattyAcid.gif" alt=""></p>
    <p><object width="75%" height="1000px" data="FAS.html"></object></p>
    <p><object width="75%" height="1000px" data="FattyAcid.html"></object></p>
    <p><img src="Figures/Fatty%20Acid.svg" alt="Fatty Acid" title="Fatty Acid"></p>
    <h2 id="15-amino-acid-metabolism">1.5. <strong>Amino acid Metabolism</strong></h2>
    <p><object width="75%" height="1000px" data="AA.html"></object></p>
    <h3 id="151-amino-acid-catabolism-and-urea-cycle">1.5.1. Amino Acid Catabolism and Urea Cycle</h3>
    <p><img src="Figures/Amino%20Acid%20Catabolism%20and%20Urea%20Cycle.png" alt="Amino Acid Catabolism and Urea Cycle"
            title="Amino Acid Catabolism and Urea Cycle"></p>
    <h3 id="152-metabolism-of-amino-acid-carbon-skeletons">1.5.2. Metabolism of Amino Acid Carbon Skeletons</h3>
    <p><img src="Figures/Metabolism%20of%20Amino%20Acid%20Carbon%20Skeletons.png"
            alt="Metabolism of Amino Acid Carbon Skeletons"></p>
    <h2 id="16-nucleotide-metabolism">1.6. <strong>Nucleotide Metabolism</strong></h2>
    <p><object width="75%" height="1000px" data="Nucleotide.html"></object></p>
    <h3 id="161-nucleotide-metabolism">1.6.1. Nucleotide Metabolism</h3>
    <p><img src="Figures/Nucleotide%20Metabolism.png" alt="Nucleotide Metabolism"></p>
    <h2 id="17-reference">1.7. <strong>Reference</strong></h2>
    <pre><code><code><div>Heirendt L, Arreckx S, Pfau T, et al. Creation and analysis of biochemical constraint-based models using the COBRA Toolbox v. 3.0[J]. Nature protocols, 2019, 14(3): 639-702.
Rowe E, Palsson B O, King Z A. Escher-FBA: a web application for interactive flux balance analysis[J]. BMC systems biology, 2018, 12(1): 84.
Brunk E, Sahoo S, Zielinski D C, et al. Recon3D enables a three-dimensional view of gene variation in human metabolism[J]. Nature biotechnology, 2018, 36(3): 272.
Nelson D L, Lehninger A L, Cox M M. Lehninger principles of biochemistry[M]. Macmillan, 2008.
Nelson D L, Cox M M. Lehninger Principles of Biochemistry Lecture Notebook[M]. Macmillan, 2004.
</div></code></code></pre>

</body>

</html>